tests/test_corona_stats.py

```diff
@@ -0,0 +1,98 @@
+import json
+
+import pytest
+from sqlalchemy.orm.exc import NoResultFound
+
+from app.database.models import CoronaStats
+from app.internal import corona_stats
+
+FAKE_DATA = [
+    {
+        "Day_Date": "2020-12-19T00:00:00.000Z",
```

Comment:
lowercase

Reply:
This is constant, why lowercase?

Reply:
Sorry if I wasn't clear, I wasn't talking about the constant but about "Day_Date" which should be "day_date".